imap/squatter.c

```diff
@@ -356,6 +356,21 @@ static int index_one(const char *name, int blocking)
         printf("Indexing mailbox %s... ", extname);
     }
 
+    if (skip_unmodified) {
```

Comment:
An interesting quirk of using `skip_unmodified` like this is that you cannot express "I want to skip the unmodified mailboxes, using a delta of 0".  No idea if this is useful, and I assume most of this implementation is just bringing back code from the old `-s` implementation?  So I'm not sure how fussy I want to be...

I think, _if_ I thought "skip the unmodified mailboxes with a delta of zero" was a useful behaviour, I might do something like:

* initialise skip_unmodified to -1
* in the `case 's':`, change it to either 60 or atoi(optarg) and complain if it's negative
* in the loop, check for `if (skip_unmodified >= 0)`

so that the sign determines whether the flag is enabled, and the magnitude is the delta.

But all that cleverness still doesn't enable negative deltas, and I still have no insight as to whether they're useful, nor whether zero is.   I dunno, what do you think?

Reply:
Zero kind of makes sense and it's simple to support, so why not. I updated the code and documentation accordingly.